EasyPost.Net35/Properties/AssemblyInfo.cs

```diff
@@ -1,5 +1,8 @@
-﻿using System.Reflection;
-using System.Runtime.CompilerServices;
+﻿// AssemblyInfo.cs
+// Copyright (c) 2022 EasyPost
+// All rights reserved.
```

Comment:
Looks like some headers remained? Could we do a search all to find any stragglers?

Reply:
Resolved in ff1fb5d